relay/kusama/src/governance/fellowship.rs

```diff
@@ -356,6 +356,8 @@ impl pallet_ranked_collective::Config<FellowshipCollectiveInstance> for Runtime
 	// - the Fellows origin
 	type ExchangeOrigin =
 		EitherOf<frame_system::EnsureRootWithSuccess<Self::AccountId, ConstU16<65535>>, Fellows>;
+	type AddOrigin = MapSuccess<Self::PromoteOrigin, ReplaceWithDefault<()>>;
+	type RemoveOrigin = Self::DemoteOrigin;
```

Comment:
We can now add custom origins for adding and removing. The current config should keep the same behaviour as before.